antarest/study/storage/variantstudy/model/command/create_binding_constraint.py

```diff
@@ -27,25 +27,28 @@
     "CreateBindingConstraint",
     "check_matrix_values",
     "BindingConstraintProperties",
+    "BindingConstraintProperties870",
+    "BindingConstraintMatrices",
 )
 
 MatrixType = List[List[MatrixData]]
 
 
-def check_matrix_values(time_step: BindingConstraintFrequency, values: MatrixType) -> None:
+def check_matrix_values(time_step: BindingConstraintFrequency, values: MatrixType, version: int) -> None:
     """
     Check the binding constraint's matrix values for the specified time step.
 
     Args:
         time_step: The frequency of the binding constraint: "hourly", "daily" or "weekly".
         values: The binding constraint's 2nd member matrix.
+        version: Study version.
 
     Raises:
         ValueError:
             If the matrix shape does not match the expected shape for the given time step.
             If the matrix values contain NaN (Not-a-Number).
     """
-    # Matrice shapes for binding constraints are different from usual shapes,
+    # Matrices shapes for binding constraints are different from usual shapes,
```

Comment:
Contrary to the French grammar rule, the noun "matrix" is singular when the following noun is plural.